internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/hadoop/TestVenicePushJob.java

```diff
@@ -1,19 +1,6 @@
 package com.linkedin.venice.hadoop;
 
-import static com.linkedin.venice.hadoop.VenicePushJob.CONTROLLER_REQUEST_RETRY_ATTEMPTS;
-import static com.linkedin.venice.hadoop.VenicePushJob.DEFER_VERSION_SWAP;
-import static com.linkedin.venice.hadoop.VenicePushJob.ENABLE_WRITE_COMPUTE;
-import static com.linkedin.venice.hadoop.VenicePushJob.INCREMENTAL_PUSH;
-import static com.linkedin.venice.hadoop.VenicePushJob.KAFKA_INPUT_BROKER_URL;
-import static com.linkedin.venice.hadoop.VenicePushJob.KAFKA_INPUT_FABRIC;
-import static com.linkedin.venice.hadoop.VenicePushJob.KAFKA_INPUT_MAX_RECORDS_PER_MAPPER;
-import static com.linkedin.venice.hadoop.VenicePushJob.KAFKA_INPUT_TOPIC;
-import static com.linkedin.venice.hadoop.VenicePushJob.KEY_FIELD_PROP;
-import static com.linkedin.venice.hadoop.VenicePushJob.SOURCE_KAFKA;
-import static com.linkedin.venice.hadoop.VenicePushJob.SUPPRESS_END_OF_PUSH_MESSAGE;
-import static com.linkedin.venice.hadoop.VenicePushJob.VALUE_FIELD_PROP;
-import static com.linkedin.venice.hadoop.VenicePushJob.VENICE_PARTITIONERS_PROP;
-import static com.linkedin.venice.hadoop.VenicePushJob.getLatestPathOfInputDirectory;
+import static com.linkedin.venice.hadoop.VenicePushJob.*;
```

Comment:
We would like to have all imports called out explicitly. Refer to this to set up your IntelliJ to not make it replace the imports with wildcards: https://linkedin-randd.slack.com/archives/G0GEB715Y/p1663714609894439